crm_project_task/wizards/crm_create_task.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2023 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class CrmCreateTAsk(models.TransientModel):
+    _name = "crm.create.task"
+    _description = "Wizard to create task"
+
+    lead_id = fields.Many2one("crm.lead")
+    task_name = fields.Char()
+    description = fields.Html()
+
+    def create_task(self):
+        project = self.env.company.crm_default_project_id
+        if not project:
+            raise UserError(
+                _(
+                    "Project not configured in settings, "
+                    "please contact with your administrator."
+                )
+            )
+        # Create task
+        task = self.env["project.task"].sudo().create(self._get_data_create(project))
+        # Messages in chatter
+        task.message_post(
+            body=_(
+                "Task created from lead/opportunity "
+                "<a href=# data-oe-model=crm.lead data-oe-id=%(lead)d>%(name)s</a>.",
```

Comment:
Link in message post is not working properly
![image](https://github.com/OCA/crm/assets/53056345/c66f1f42-f78e-47f4-bff7-e2dd5bf0782e)